groupcache.go

```diff
@@ -121,6 +122,20 @@ func RegisterNewGroupHook(fn func(*Group)) {
 	newGroupHook = fn
 }
 
+// RegisterErrLogFn registers a log func that is used for http error logging
```

Comment:
The "Fn" suffix isn't consistent with the naming elsewhere in Go in general or this file in particular. This file uses "Hook".

Also, end sentences with a period. And http should be HTTP.